spec/index.html

```diff
@@ -969,8 +969,9 @@ <h3>Algorithm</h3>
               as well as instantiating a new <a>canonical issuer</a>.</p>
             <p>After this algorithm completes,
               the <a>input blank node identifier map</a> state
-              may be used to correlate blank node identifiers
-              used in the <a>input dataset</a> to those used
+              and / or <a>canonical issuer</a> may be used to
```

Comment:
```suggestion
              and/or <a>canonical issuer</a> may be used to
```

Reply:
That text was changed, as it now includes both these things.